docs/components/copy.md

```diff
@@ -0,0 +1,183 @@
+# Copy
+
+KCopy is a component that displays provided text and allows a user to copy it to their clipboard.
+
+<KCopy :text="text" />
+
+```html
+<KCopy :text="text" />
+```
+
+## Props
+
+### text
+
+The copyable text.
+
+### badge
+
+Whether or not to display as a badge. Defaults to `false`.
+
+<KCopy
+  badge
+  truncate
+  :text="text"
+/>
+
+```html
+<KCopy
+  badge
+  truncate
+  :text="text"
+/>
+```
+
+### badgeText
+
+Text displayed before the copyable text when `badge` is true.
+
+<KCopy
+  badge
+  badge-text="Id"
+  :text="text"
+/>
+
+```html
+<KCopy
+  badge
+  badge-text="Id"
+  :text="text"
+/>
+```
+
+### truncate
+
+Whether or not the text should be truncated. Defaults to `false`.
+
+<KCopy
+  truncate
+  :text="text"
+/>
+
+```html
+<KCopy
+  truncate
+  :text="text"
+/>
+```
+
+### truncationLimit
+
+Number of characters to truncate at. Defaults to `8`.
+
+<KCopy
+  truncate
+  :text="text"
+  :truncation-limit="15"
+/>
+
+```html
+<KCopy
+  truncate
+  :text="text"
+  :truncation-limit="15"
+/>
+```
+
+### copyTooltip
```

Comment:
![image](https://github.com/Kong/kongponents/assets/2229946/b02033cb-97ca-471b-83db-38eee2c159f6)
question: When I click to copy, the tooltip doesn't change when this prop is provided. Is that desired?

Reply:
Fixed.